File: src/main/java/com/manson/domain/fed76/ReviewRatingResponse.java
package com.manson.domain.fed76;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.manson.domain.AbstractObject;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(callSuper = false)
@ToString(callSuper = true)
public class ReviewRatingResponse extends AbstractObject {

    @JsonProperty("@type")
    private String type;
    private String bestRating;
    private String ratingValue;
    private String worstRating;
}

File: src/main/kotlin/com/manson/domain/fed76/pricing/ReviewRatingResponse.kt
package com.manson.domain.fed76.pricing

import com.manson.AbstractObject

open class ReviewRatingResponse : AbstractObject() {
    var bestRating: String = "EMPTY"
    var ratingValue: String = "EMPTY"
    var worstRating: String = "EMPTY"
    override fun toString(): String {
        return "ReviewRatingResponse(bestRating='$bestRating', ratingValue='$ratingValue', worstRating='$worstRating')"
    }

}